Plugin/UIScopeLabel.php

```diff
@@ -0,0 +1,34 @@
+<?php
+namespace AvS\ScopeHint\Plugin;
+
+use Magento\Eav\Api\Data\AttributeInterface;
+use Magento\Framework\Stdlib\ArrayManager;
+use Magento\Catalog\Ui\DataProvider\Product\Form\Modifier\Eav;
```

Comment:
Could you introduce an alias for this? `Eav` is not self-explaining, but `EavFormModifier` would be.

Reply:
Done.